tools/packaging/kata-deploy/scripts/kata-deploy.sh

```diff
@@ -427,6 +432,7 @@ function main() {
 	echo "* CREATE_RUNTIMECLASSES: ${CREATE_RUNTIMECLASSES}"
 	echo "* CREATE_DEFAULT_RUNTIMECLASS: ${CREATE_DEFAULT_RUNTIMECLASS}"
 	echo "* ALLOWED_HYPERVISOR_ANNOTATIONS: ${ALLOWED_HYPERVISOR_ANNOTATIONS}"
+	echo "* SELINUX: ${SELINUX}"
```

Comment:
@ldoktor, we don't have to print this one here, as the environment variable is not coming from the daemonset sepc file.
I'd print an info, though, in the part I mentioned above.

Reply:
sure, I'll move it to the selinux variable setter (once we get a conclusion on which way to chose)